packages/docs/docs/features/storybook-integration.md

```diff
@@ -0,0 +1,24 @@
+---
+id: storybook-integration
+title: Storybook integration
+sidebar_position: 7
+---
+
+Radon IDE seamlessly integrates with Storybook. The extension automatically detects Storybook stories and
```

Comment:
and?

Reply:
fixed in [ee9c612](https://github.com/software-mansion/radon-ide/pull/689/commits/ee9c612f824d826cdb220e9d5ad545542ba1f265)